apps/src/templates/projects/projectsModule.js

```diff
@@ -0,0 +1,30 @@
+/** @file Redux actions and reducer for the Projects Gallery */
+import {registerReducers} from '@cdo/apps/redux';
+
+// Actions
+const TOGGLE_GALLERY = 'Projects/TOGGLE_GALLERY';
+
+const isPublic = window.location.pathname.startsWith('/projects/public');
+
+// Reducer
+function pageLocation(state, action) {
+  const defaultState = isPublic ? 'PUBLIC' : 'PRIVATE';
```

Comment:
Thinking about this some more, I think I would move the isPublic check out of this module and have a caller set the state based on window.location. This module itself should not depend on window.location.